typings/resin-sdk.d.ts

```diff
@@ -857,12 +880,81 @@ declare namespace ResinSdk {
 					): Promise<void>;
 					remove(uuidOrId: string | number, tagKey: string): Promise<void>;
 				};
+				configVar: {
+					getAllByDevice(
+						uuidOrId: string | number,
+						options?: PineOptionsFor<DeviceConfigVariable>,
+					): Promise<DeviceConfigVariable[]>;
+					set(
+						uuidOrId: string | number,
+						key: string,
+						value: string,
+					): Promise<void>;
+					get(
+						uuidOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+					remove(
+						uuidOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+				};
+				envVar: {
+					getAllByDevice(
```

Comment:
I don't want to add `getAll` because I don't think there's many use cases for getting _all_ variables everywhere, and the existing methods take options anyway, but I have added `getAllByApplication` to service vars, device env vars, device config vars and device service vars. That should cover everything you need to do things like the UI is doing (show app level & overrides in one page) with only two queries.